src/main/java/com/qaprosoft/carina/demo/gui/pages/AutomationPracticeAddedProductModalPage.java

```diff
@@ -33,4 +34,12 @@ public AutomationPracticeShoppingCartSummary clickProceedToCheckOutButton() {
         proceedToCheckOutButton.click();
         return new AutomationPracticeShoppingCartSummary(driver);
     }
+
+    public boolean isProceedToCheckOutButtonPresent() {
+        return proceedToCheckOutButton.isElementPresent() ;
+    }
+
+    public boolean isCartNotEmpty() {
```

Comment:
not sure that it's correct/clear name for method because it is not a cart. If I wanted to verify that product is shown on this modal I would get the product name that I added to cart in previous steps and create here a method
public String getProductNameOnModal()
And in the test check if product name equals on modal and in search results